### ISSUE TITLE
Missing alt tag for filmstrip image on /ProgramVideos page

### ISSUE THREAD

**Issue by JoeProgrammer88** [SpeakingInBits/PC2#280]
Related to #40 
The page /ProgramVideos has an image with a filmstrip and is missing the alt tag. This must be addressed for accessbility

Helpful information
https://accessibilityinsights.io/info-examples/web/image-alt/

**Comment by JoeProgrammer88** [comment]
Closed in #287
